test/upload-test-images.sh

```diff
@@ -27,7 +27,7 @@ function upload_test_images() {
 
   # ko resolve is being used for the side-effect of publishing images,
   # so the resulting yaml produced is ignored.
-  ko resolve ${tag_option} -RBf "${image_dir}" > /dev/null
+  ko resolve --strict ${tag_option} -RBf "${image_dir}" > /dev/null
```

Comment:
The path for the test images in eventing was not updated in that PR. So this won't work..

Reply:
removed, let's see now